spec/factories/categories.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :category, :class => Fagus::Category do
+    description { "#{reference} - #{name}" }
```

Comment:
You have `is_active` declared in the schema. You will probably need to add the field into the factory.